guide/14-deep-learning/object-detection.ipynb

```diff
@@ -69,7 +69,7 @@
        "                        </a>\n",
```

Comment:
<p>Title could be changed as --  <strong>Model Deployment</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>